### PR TITLE
Added ability to search by postal code to get postal town, county and coordinates

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

File: src/main/kotlin/com/hedvig/lookupservice/config/PostalConfig.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                            municipality = municipality,
+                            city = city,
+                            latitude = latitude,
+                            longitude = longitude
+                        )
+                    )
+                }
+                postalRepository.saveAll(postals)
+                logger.info("Added ${postals.size} postal codes to $country")
+                postals = mutableListOf() // TODO: Discuss other optimization (these lists are really long 16k for Sweden only) e.g. using postalRepository.save straight away?
```
Comment (FredrikAreschoug) [inline src/main/kotlin/com/hedvig/lookupservice/config/PostalConfig.kt]: Regarding the todo either way works :) but I prefer saving straight away. `postals = mutableListOf()` is not needed it will go out of scope and garbage collected after this line

File: src/main/kotlin/com/hedvig/lookupservice/config/PostalConfig.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            }
+        } catch (exception: Exception) {
+            logger.error("Exception when filling postal database (exception=$exception)")
```
Comment (palmenhq) [inline src/main/kotlin/com/hedvig/lookupservice/config/PostalConfig.kt]: `logger.error("exception when...", exception)` :) that way the stack trace will remain

File: src/main/kotlin/com/hedvig/lookupservice/postal/model/Postal.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package com.hedvig.lookupservice.postal.model
+
+import java.math.BigDecimal
+import javax.persistence.*
+
+@Entity
+@IdClass(PostalId::class) // FIXME: Still need equals and hash if we use composite key?
```
Comment (palmenhq) [inline src/main/kotlin/com/hedvig/lookupservice/postal/model/Postal.kt]: i think yes

File: src/main/kotlin/com/hedvig/lookupservice/config/PostalConfig.kt [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        postalRepository.deleteAll()
+        try {
+            var postals = mutableListOf<Postal>()
+            Country.values().forEach { country ->
+                val fileReader = BufferedReader(FileReader("$postalCodesLocation/$country.csv"))
+                val header = fileReader.readLine()
+                logger.info("Filling data of country=$country data with header=$header")
+                val lines = fileReader.readLines()
+                for (line in lines) {
+                    val tokens = line.split(',')
```
Comment (palmenhq) [inline src/main/kotlin/com/hedvig/lookupservice/config/PostalConfig.kt]: i'm really scared of parsing csv by hand. Its probably fine for data this simple but for future cases it could be good to use a "proper" library. Here's some good reading on it https://dotnetcoretutorials.com/2018/08/04/csv-parsing-in-net-core/

TLDR;
there's a lot of strange corner cases where this solution would trip
